crates/ruff_linter/Cargo.toml

```
@@ -76,6 +76,7 @@ insta = { workspace = true, features = ["filters", "json", "redactions"] }
 test-case = { workspace = true }
 # Disable colored output in tests
 colored = { workspace = true, features = ["no-color"] }
+tempfile.workspace = true
```

```suggestion
tempfile = { workspace = true }
```

I don't know why we use the long form, I think @charliermarsh migrated it some time ago but I prefer to keep using a consistent format.